app/jobs/trainees/create_from_dttp_job.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Trainees
+  class CreateFromDttpJob < ApplicationJob
+    queue_as :default
+
+    def perform
+      return unless FeatureService.enabled?("import_trainees_from_dttp")
+
+      Dttp::Trainee.joins(:provider).includes(:placement_assignments).unprocessed.each do |dttp_trainee|
```

Comment:
I think we'd be better doing a job that just creates a single trainee and then letting the calling code handle this. Because:
* we can scale the workers out then and do these in parallel
* we almost certainly won't want to import all unprocessed in one go

Reply:
Changed in 49342cd4a1943343a6b85b5bcd1be093bf550813